hugolib/page_bundler.go

```diff
@@ -14,13 +14,13 @@
 package hugolib
 
 import (
+	"context"
 	"fmt"
 	"math"
 	"runtime"
 
 	// Use this until errgroup gets ported to context
 	// See https://github.com/golang/go/issues/19781
```

Comment:
If you could also remove the now superflous comments above, I'll merge.

Reply:
Done and combined into one commit.